cromwellApiClient/src/main/scala/cromwell/api/model/CromwellStatus.scala

```diff
@@ -0,0 +1,9 @@
+package cromwell.api.model
```

Comment:
I dunno, this was directly ported from Centaur so I'm inclined to leave it like this for now... if you have strong aesthetic objections, feel free to shuffle them around afterwards...